src/app/QuickStarts/quickstarts/automated-rules-quickstart.tsx

```diff
@@ -78,86 +75,74 @@ In this quick start, you will use [APP] to create an automated rule that will st
-`,
+1. In the [APP] console navigation bar, click [Automated Rules]{{highlight nav-automatedrules-tab}}.
+2. Click [Create]{{highlight create-rule-btn}}.
+        `,
         review: {
           instructions: '#### Verify that you see the Automated Rules creation form.',
-          failedTaskHelp: 'If you do not see the creation form, follow the previous steps again.',
+          failedTaskHelp:
+            'If you do not see the navigation bar, click the [menu button]{{highlight nav-toggle-btn}} on the masthead.',
         },
       },
       {
         title: 'Fill out the Automated Rule form',
         description: `
-The Automated Rule creation form has several fields that you can fill out to create a new rule. Each field has helper text that explains what the field is for.
-
-**The most important field is the [Match Expression]{{highlight rule-matchexpr}} field.** This field is used to match one or more target JVMs. The match expression is a Java-like code snippet that is matched against each target JVM. For example, if you wanted to match all discovered target JVMs, try using the match expression: \`true\`{{copy}}.
+To create a new rule, you'll use the Automated Rule creation form which has multiple fields with helpful text to guide you. Each field is labeled to explain its purpose and help you fill out the form correctly.
 
-[Use the [Match Expression Visualizer]{{highlight match-expr-card}} to test your match expression against the target JVMs that are currently discovered by [APP]. Matched targets will be unfaded in the Graph view, and listed in the List View.]{{admonition tip}}
+**The most important field is the [Match Expression]{{highlight rule-matchexpr}} field.** This field is used to match one or more target JVMs. The match expression is a Java-like code snippet that is matched against each target JVM.
 
-1. Select a target JVM from the [Target]{{highlight rule-target}} dropdown menu.
-2. Enter a [Match Expression]{{highlight rule-matchexpr}} for the rule. Try using the match hint to help you create a match expression.
-3. Note the target JVM details code block in the tester. These details can be used in the match expression.
+[Use the [Match Expression Visualizer]{{highlight match-expr-card}} to test your match expression against the target JVMs currently discovered by [APP]. Any matched targets will appear unfaded in the Graph view and will be listed in the List view.]{{admonition tip}}
 
 **To create a new rule, you must fill out the following required fields:**
 1. Enter a name for the rule in the [Name]{{highlight rule-name}} field.
-2. Enter a [Match Expression]{{highlight rule-matchexpr}} for the rule.
-3. Select the [Event Template]{{highlight rule-evt-template}} you want to use for the rule.
+2. Enter a [Match Expression]{{highlight rule-matchexpr}}. Try the [match hint]{{highlight rule-matchexpr-help}} to help you create a match expression or use the expression \`true\`{{copy}}.
```

Comment:
From guideline:

> Clearly distinguish between a user action and additional information on product functionality.
> Additional information:
>  To look at data in a specific time frame, you can change the time range of the dashboard.

How about?

```
To see an example match expression, click [match hint]{{highlight rule-matchexpr-help}}.
```

We might not need to call out `true` here if we elaborate more on the match expression previously. 

Also, could we pull the tips about match expression visualizer after these steps?

Reply:
Also, `Enter a [Match Expression]{{highlight rule-matchexpr}}` should be `Enter an expression in the [Match Expression]{{highlight rule-matchexpr}} field.`